src/templates/components/imageAndStaticMap/index.tsx

```diff
@@ -14,18 +13,7 @@ export const ImageAndStaticMap: React.FC<ImageAndStaticMapProps> = ({
 }) => {
   return (
     <div className="usa-width-one-third inline-table-helper vads-u-order--first mobile-lg:vads-u-order--initial vads-u-margin-bottom--2 vads-u-margin-left--auto facility">
-      <div
-        onClick={() =>
-          recordEvent({ event: 'image-click', 'facility-name': facilityId })
-        }
-        style={{ cursor: 'pointer' }}
-      >
-        <MediaImage
-          {...image}
-          imageStyle="2_1_large"
-          className="facility-img"
-        />
-      </div>
+      <MediaImage {...image} imageStyle="2_1_large" className="facility-img" />
```

Comment:
Approved too quick...Laura called this out, but minor thing that you can just merge after fixing, this should be `3_2_medium_thumbnail`